entries/u/umbrella.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Umbrella": {
+    "domain": "umbrella.com",
+    "url": "https://umbrella.cisco.com",
```

Comment:
Since `https://umbrella.com` already redirects to ` https://umbrella.cisco.com`, the url element isn't needed.
```suggestion
```